src/Content/Text/BBCode.php

```diff
@@ -1220,6 +1220,16 @@ public static function cleanPictureLinks($text)
 		return $return;
 	}
 
+	public static function removeLinks(string $bbcode)
+	{
+		$bbcode = preg_replace("/\[img\=(.*?)\](.*?)\[\/img\]/ism", ' $1 ', $bbcode);
+		$bbcode = preg_replace("/\[img.*?\[\/img\]/ism", ' ', $bbcode);
+
+		$bbcode = preg_replace("/\[url=[^\[\]]*\](.*)\[\/url\]/Usi", ' $1 ', $bbcode);
+		$bbcode = preg_replace('/[@!#]?\[url.*?\[\/url\]/ism', '', $bbcode);
```

Comment:
Hmm, this would convert all tag links into tag strings, I have to suggest something else.
```suggestion
		$bbcode = preg_replace('/[@!#]\[url\=.*?\].*?\[\/url\]/ism', '', $bbcode);
		$bbcode = preg_replace("/\[url=[^\[\]]*\](.*)\[\/url\]/Usi", ' $1 ', $bbcode);
		$bbcode = preg_replace('/[@!#]?\[url.*?\[\/url\]/ism', '', $bbcode);
```